apps/api/src/main/resources/application.properties

```diff
@@ -1,3 +1,6 @@
 server.port = 5000
 spring.datasource.driver-class-name = org.postgresql.Driver
-spring.config.import = optional:file:api.env[.properties]
\ No newline at end of file
+spring.config.import = optional:file:api.env[.properties]
+spring.security.oauth2.client.registration.cognito.client-id=${COGNITO_CLIENT_ID}
+spring.security.oauth2.client.registration.cognito.client-secret=${COGNITO_CLIENT_SECRET}
+spring.security.oauth2.client.provider.cognito.issuer-uri=https://cognito-idp.eu-west-1.amazonaws.com/eu-west-1_dw3OkqpZT
```

Comment:
Place the three things above directly into the api.env (and swap out the ${} values)